app/controllers/public/index.js

```diff
@@ -6,10 +6,14 @@ export default Controller.extend({
 
   speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
```

Comment:
Single array is a good option but that might cause us some problems. we are checking isFeatured attribute to change classes of the HTML in the `speaker-item` which is commonly used by Speakers and Featured Speakers, this will lead to change of layout for featured speaker in the Speakers section too.